cvat-ui/src/reducers/index.ts

```diff
@@ -854,6 +854,12 @@ export enum ColorBy {
     LABEL = 'Label',
 }
 
+export enum SizeType {
+    WIDTH = 'width',
+    HEIGHT = 'height',
+    LENGTH = 'length',
+}
```

Comment:
We only use this enum in two component files (like SizeParams interface), I am not sure it will be necessary in other components, and thus, I do not think we need to keep it in reducers. 

Maybe it is a good place for types used somehow in the app storage, but not for any

Reply:
Moved to `object-item-details`